sunpy/map/sources/tests/test_gong_synoptic_source.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+import astropy.units as u
+
+from sunpy.data.test import get_dummy_map_from_header, get_test_filepath
+from sunpy.map.sources.gong import GONGSynopticMap
+
+
+@pytest.fixture
+def gong_synoptic():
+    return get_dummy_map_from_header(get_test_filepath('gong_synoptic.header'))
+
+
+def test_fitstoGONGSynoptic(gong_synoptic):
+    """Tests the creation of GongSynopticMap using FITS."""
+    assert isinstance(gong_synoptic, GONGSynopticMap)
+
+
+def test_is_datasource_for(gong_synoptic):
+    """Test the is_datasource_for method of GongSynopticMap."""
+    assert gong_synoptic.is_datasource_for(gong_synoptic.data, gong_synoptic.meta)
+
+
+def test_observatory(gong_synoptic):
+    """Tests the observatory property of the GongSynopticMap object."""
+    assert gong_synoptic.observatory == "NSO-GONG"
+
+
+def test_measurement(gong_synoptic):
+    """Tests the measurement property of the GongSynopticMap object."""
+    assert gong_synoptic.measurement == 676.8
```

Comment:
It's a bit worrying to me that the measurement (which here is just the wavelength) is being returned without a unit. In the above test header, the unit (nm) is included in the key comment, but it does not seem to be being parsed correctly. This is out of scope of this PR though so I'm OK merging and fixing this later.

Reply:
See #7245. If that is merged, we can rebase this PR and then change the test to check for `676.8 * u.nm`